ci/kind/test-e2e-kind.sh

```diff
@@ -128,18 +138,41 @@ function run_test {
   echo "creating test bed with args $args"
   eval "timeout 600 $TESTBED_CMD create kind --antrea-cni false $args"
 
+  yamlFile=""
   if $coverage; then
-      $YML_CMD --kind --encap-mode $current_mode $manifest_args | docker exec -i kind-control-plane dd of=/root/antrea-coverage.yml
+      $YML_CMD --kind --encap-mode $current_mode $manifest_args | tee antrea-coverage.yml |docker exec -i kind-control-plane dd of=/root/antrea-coverage.yml
       $FLOWAGGREGATOR_YML_CMD --coverage | docker exec -i kind-control-plane dd of=/root/flow-aggregator-coverage.yml
+      yamlFile="antrea-coverage.yml"
   else
-      $YML_CMD --kind --encap-mode $current_mode $manifest_args | docker exec -i kind-control-plane dd of=/root/antrea.yml
+      $YML_CMD --kind --encap-mode $current_mode $manifest_args | tee antrea.yaml | docker exec -i kind-control-plane dd of=/root/antrea.yml
       $FLOWAGGREGATOR_YML_CMD | docker exec -i kind-control-plane dd of=/root/flow-aggregator.yml
+      yamlFile="antrea.yml"
   fi
   sleep 1
+  # extra configmap yaml from antrea.yml
```

Comment:
Maybe we can have a `antrea-configmap.yml` under `build/yamls` and let `generate-manifest.sh` combines these yaml files into `antrea.yml`. Then, there's no need to have such complex script here to select configmap part from `antrea.yml`.

Reply:
I added a new option "--configmap-only" to allow user to call generate-manifest.sh to generate `antrea-config.yml` file only in `test-e2e-kind.sh` without add a new `antrea-configmap.yml` under `build/yamls` considering e2e is the only case to use `antrea-config.yml`, we can change the Makefile in the future to add new file if there are more cases.